themes/default/content/blog/slack-integration/index.md

```diff
@@ -0,0 +1,397 @@
+---
+title: "Announcing Slack and Deployment Notifications in Pulumi Cloud"
+
+allow_long_title: true
+# The date represents the post's publish date,
+# and by default corresponds with the date this file was generated.
+# Posts with future dates are visible in development,
+# but excluded from production builds.
+# Use the time and timezone-offset portions of of this value
+# to schedule posts for publishing later.
+date: 2023-05-26T11:07:30-07:00
+
+# Use the meta_desc property to provide a brief summary
+# (one or two sentences) of the content of the post,
+# which is useful for targeting search results or social-media previews.
+# This field is required or the build will fail the linter test.
+# Max length is 160 characters.
+meta_desc: Pulumi Cloud now has an easy to set up Slack integration, Pulumi Deployments notifications and event filtering.
+
+# The meta_image appears in social-media previews and on the blog home page.
+# A placeholder image representing the recommended format, dimensions and aspect ratio
+# has been provided for you.
+meta_image: meta.png
+
+# At least one author is required.
+# The values in this list correspond with the `id` properties
+# of the team member files at /data/team/team.
+# Create a file for yourself if you don't already have one.
+authors:
+    - meagan-cojocar
+    - komal-ali
+
+# At least one tag is required.
+# Lowercase, hyphen-delimited is recommended.
+tags:
+    - features
+    - pulumi-cloud
+
+# See the blogging docs at https://github.com/pulumi/pulumi-hugo/blob/master/BLOGGING.md.
+# for additional details,
+# and please remove these comments before submitting for review.
+---
+
+Today, we are excited to introduce a set of improvements to [Pulumi Cloud Webhooks](/docs/pulumi-cloud/webhooks), designed to streamline your deployment notifications to where you already spend your time, enabling faster response times to critical issues. We are also announcing two new improvements to our webhooks feature: Pulumi Deployments events and fine-grained filtering.
```

Comment:
```suggestion
Today, we are excited to introduce a set of improvements to [Pulumi Cloud Webhooks](/docs/pulumi-cloud/webhooks), designed to streamline your deployment notifications to where you already spend your time, enabling faster response times to critical issues. We are also announcing two new improvements to our webhooks feature: Pulumi Deployments events and fine-grained event filtering.
```

Reply:
This sentence should also mention Slack specifically, I can't quite figure out the wordsmithing to get it in there.